dashboard/views.py

```diff
@@ -313,7 +313,8 @@ def grade_distribution(request, course_id=0):
         from user where course_id=%(course_id)s and enrollment_type='StudentEnrollment';
                     """
     df = pd.read_sql(grade_score_sql, conn, params={"current_user": current_user, 'course_id': course_id})
-    if df.empty or df['current_grade'].isnull().all():
+    if df.empty or df['current_grade'].isnull().all() or df.count().current_grade < 6:
```

Comment:
Could we simplify this condition to just `df['current_grade'].count() < 6`? If we always have a `current_grade` column in the DataFrame we get back from the SQL query, this should catch any of the cases, I think?

Reply:
Also, do we want the threshold to be higher than 6? I can imagine in the short window where some students have grades and others don't, we could have similar privacy concerns to what we had with outliers (i.e. bars with 1 student at the top or bottom)? @jennlove-um, any thoughts?